sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -630,7 +630,7 @@ public void init(CosmosClientMetadataCachesSnapshot metadataCachesSnapshot, Func
                     this,
                     this.connectionPolicy.getPreferredRegions());
             clientTelemetry.init().thenEmpty((publisher) -> {
-                logger.info(
+                logger.warn(
```

Comment:
why change to warn here

Reply:
I want the vmId - in the c logs with Warn there was nowhere the vmId emitted